### PR TITLE
Kujira GHOST

### PR DESCRIPTION
**Pull request by codehans**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):

Ghost 

##### Twitter Link:

https://twitter.com/TeamKujira

##### Website Link:

https://ghost.kujira.app/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
ghost
ghost

##### Current TVL:

$5,227

##### Chain:

kujira

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)

kujira

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)

kujira

##### Short Description (to be shown on DefiLlama):

Ghost is the native lending and borrowing market on the Kujira blockchain

##### Category (full list at https://defillama.com/categories) *Please choose only one:

Lending

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):

https://github.com/Team-Kujira/oracle-price-feeder

##### forkedFrom (Does your project originate from another project):

none

##### methodology (what is being counted as tvl, how is tvl being calculated):

lender deposits - borrower collateral will follow when live

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ghost exports TVL: 
        
 
 
    kujira                    10.70 k
    
    total                    10.70 k

File: projects/helper/chain/cosmos.js
Comment (g1nt0ki) [inline projects/helper/chain/cosmos.js:1]: sorry, can you revert back the formatting changes on this file.

**Comment by llamatester** [comment]
The adapter at projects/ghost exports TVL: 
        
 
 
    kujira                    1.68 M
    
    total                    1.68 M

File: projects/helper/chain/cosmos.js
Comment (codehans) [reply projects/helper/chain/cosmos.js:1]: @g1nt0ki yup done

**Comment by llamatester** [comment]
The adapter at projects/ghost exports TVL: 
        
 
 
    kujira                    837.80 k
    
    total                    837.80 k

File: projects/helper/chain/cosmos.js
Comment (g1nt0ki) [reply projects/helper/chain/cosmos.js:1]: @codehans thanks for reverting it, I am just tired of reviewing more lines than necessary and having to be paranoid that something is  slipped in.

Comment (codehans) [reply projects/helper/chain/cosmos.js:1]: All good. Yeah completely understand. Was an overzealous prettier config at my end

**Comment by llamatester** [comment]
The adapter at projects/ghost exports TVL: 
        
 
 
    kujira                    829.24 k
    
    total                    829.24 k

**Pull request merged**